tensorflow/core/kernels/resize_bilinear_op_gpu.cu.cc

```diff
@@ -25,12 +25,78 @@ limitations under the License.
 #include "tensorflow/core/platform/types.h"
 #include "tensorflow/core/util/gpu_kernel_helper.h"
 
+// auxilary 16-byte datatype for ResizeBilinearKernel_faster
+// the fields are not important. The only purpose of this is to read 16 bytes
+// from GPU gloal memory
+struct four_floats{
+  float a;
+  float b;
+  float c;
+  float d;
+};
+
 namespace tensorflow {
 
 typedef Eigen::GpuDevice GPUDevice;
 
 namespace {
 
+template <typename T, int C_UNROLL>
+__global__ void ResizeBilinearKernel_faster(const int num_channel_thread, const T* __restrict__ images,
```

Comment:
How about calling this ResizeBilinearKernelUnrolled or something like that?

Reply:
Done